README.md

```diff
@@ -10,35 +10,38 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
 
 ### 💡💥❓ How things work
 
-Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request, I create a new branch that is called something like first-timers-only-typo-in-title. GitHub will notify the "first-timers-only issue bot" about the new branch using Webhooks. Currently, the bot will see that a new branch was created starting with "first-timers-only" and it will create a new issue at https://github.com/hoodiehq/camp/issues/new based on a template and assign the first-timers-only and the up-for-grabs label. If the commit body can be used to add some context information. If left empty, the 🤔 What you will need to know section of the issue will simply say "Nothing :)". Otherwise the commit body will be used there.
+Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request which is time-consuming, I can simply create a new branch that is called something like **first-timers-only-typo-in-title.** GitHub will then notify the "first-timers-only issue bot" about the new branch using Webhooks. The bot is listening to any new branch starting with **first-timers-only** and it will create a new issue on your repo. It is currently setup to the Hoodie repo with a template and assigns the first-timers-only and up-for-grabs labels but can be modified by the developer. The commit body can be used to add some context information and if left empty, the 🤔 **What you will need to know** section of the issue will simply say "Nothing :)".
+
+
+### 😮 🙌 👀 🎉 Use Our Bot!
+
+<table>
+    <tr>
+        <th>Steps</th>
+        <th>Example</th>
+    </tr>
+    <tr>
+        <td>1) <a href="https://github.com/apps/first-timers-bot">Install App</a> on a repo of your choice</td>
+    </tr>
+    <tr>
+        <td>2) Click on the file you want to edit.</td>
+        <td><img src="/assets/Edit.png?raw=true"></td>
+    </tr>
+    <tr>
+        <td>3) Make the change and write your commit message under <b>Commit changes</b>.  Make sure to check <i>Create a new branch</i> at the bottom and begin the title of the branch with <b>first-timers-only</b> for example.</td>
+        <td><img src="/assets/Committing-Branch.png?raw=true"></td>
+    </tr>
+    <tr>
+        <td>4) Click on the <b>issues</b> tab and notice your issue was created with your change and commit message. The contributor would then follow the steps on the issue message.</td>
+        <td><img src="/assets/Issue-Generated.png?raw=true"</td>
+</tr>
```

Comment:
nitpick: indentation :)

Reply:
The branch to be picked up just needs to start with "first-timers-"
Check out line 7 of server.js.